pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/NamespacesBase.java

```diff
@@ -1787,6 +1787,69 @@ protected void internalSetSchemaAutoUpdateCompatibilityStrategy(SchemaAutoUpdate
             "schemaAutoUpdateCompatibilityStrategy");
     }
 
+    protected boolean internalGetSchemaValidationEnforced() {
+        validateSuperUserAccess();
+        validatePoliciesReadOnlyAccess();
+
+        try {
+            Stat nodeStat = new Stat();
+            final String path = path(POLICIES, namespaceName.toString());
+            byte[] content = globalZk().getData(path, null, nodeStat);
+            Policies policies = jsonMapper().readValue(content, Policies.class);
+            log.info("[{}] Successfully get schemaValidationEnforced configuration: namespace={}, value={}",
+                    clientAppId(), namespaceName, policies.schema_validation_enforced);
+            return policies.schema_validation_enforced;
+
+
+        } catch (KeeperException.NoNodeException e) {
+            log.warn("[{}] Failed to get schemaValidationEnforced configuration for namespace {}: does not exist",
+                    clientAppId(), namespaceName);
+            throw new RestException(Status.NOT_FOUND, "Namespace does not exist");
+        } catch (KeeperException.BadVersionException e) {
+            log.warn("[{}] Failed to get schemaValidationEnforced configuration for namespace {}: concurrent modification",
+                    clientAppId(), namespaceName);
+            throw new RestException(Status.CONFLICT, "Concurrent modification");
+        } catch (RestException pfe) {
+            throw pfe;
+        } catch (Exception e) {
+            log.error("[{}] Failed to get schemaValidationEnforced configuration for namespace {}",
+                    clientAppId(), namespaceName, e);
+            throw new RestException(e);
+        }
+    }
+
+    protected void internalSetSchemaValidationEnforced(boolean schemaValidationEnforced) {
```

Comment:
I would suggest you using mutatePolicy rather than copying the code from other methods again.

```
protected void internalSetSchemaValidationEnforced(boolean schemaValidationEnforced) {
        validateSuperUserAccess();
        validatePoliciesReadOnlyAccess();

        mutatePolicy((policies) -> {
                policies. schema_validation_enforced = schemaValidationEnforced;
                return policies;
            }, (policies) -> policies. schema_validation_enforced,
            "schemaValidationEnforced");
    }
```